CollateralAccount/CollateralAccount.md

```diff
@@ -0,0 +1,106 @@
+---
+CIP: ?
+Title: Collateral Account for HD Wallets
+Authors: Sebastien Guillemot <seba@dcspark.io>
+Comments-URI: https://forum.cardano.org/t/collateral-account-derivation/65879
+Status: Draft
+Type: Standards
+Created: 2021-06-29
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This document describes using a separate derivation path to ensure there is always sufficient collateral to execute UTXO smart contracts
+
+# Motivation
+
+Collateral input | As of Alonzo, transactions that call non-native smart contracts are required to put up collateral to cover the potential cost of smart contract execution failure. Inputs used as collateral has the following properties:
+
+- Cannot contain any tokens (only ADA)
+- Cannot be a script address
+- Must be a UTXO input
+- Must be at least some percentage of the fee in the tx (concrete percentage decided by a protocol parameter)
+- Can be the same UTXO entry as used in regular tx input
+- Is consumed entirely (no change) if the contract execution fails during phase 2 validation
+
+We therefore need a way for the wallet to handle picking which inputs to use for collateral
+
+## Naive solution
+
+The naive solution would do the following steps
+1. Add any regular input that satisfies the constraints as collateral as well
```

Comment:
'Regular input' means ADA correct? From the above statement: "Cannot contain any tokens (only ADA)" or is this referencing using any Cardano Native token that is involved in the transaction?

Reply:
This was referring to UTXO input that isn't a collateral input. I don't think we have a name for non-collateral inputs so I just used "regular" but maybe I should just say "non-collateral input"

Reply:
I'd change it to 'Non-Collateral input' to clear up possible confusion.